setup.py

```diff
@@ -68,21 +68,20 @@ def pkgconfig(*packages, **kw):
 
     if sys.platform == 'win32':
 
-        IPOPT_INCLUDE_DIRS = ['include_mt/coin', np.get_include()]
-        IPOPT_LIBS = ['Ipopt39', 'IpoptFSS']
-        IPOPT_LIB_DIRS = ['lib_mt/x64/release']
-        IPOPT_DLL = ['Ipopt39.dll', 'IpoptFSS39.dll']
+        IPOPT_INCLUDE_DIRS = ['include/coin', np.get_include()]
+        IPOPT_LIBS = ['Ipopt-vc8', 'IpOptFSS', 'IpOpt-vc10']
+        IPOPT_LIB_DIRS = ['lib/x64/ReleaseMKL']
+        IPOPT_DLL = ['IpOptFSS.dll', 'Ipopt-vc8.dll', 'IpOpt-vc10.dll', 'libiomp5md.dll', 'msvcp100.dll', 'msvcr100.dll']
```

Comment:
Do you know what version of IPOPT made these changes? It could be helpful to add that we only support ipopt >= some version for windows in the README.

Reply:
From the official builds available [here](https://www.coin-or.org/download/binary/Ipopt/), the directory is organized this way starting Ipopt 3.10.1.

Reply:
Can you note this in the README? i.e. that for Windows we only support Ipopt >= 3.10.1.

Reply:
Added in latest commit.